app/views/dradis/plugins/projects/export/_plugin_description.html.erb

```diff
@@ -0,0 +1,10 @@
+<li class="nav-item">
```

Comment:
It seems that this is more complicated than it needs to be?

https://github.com/dradis/dradis-calculator_cvss/blob/master/app/views/dradis/plugins/calculators/cvss/issues/_show-tabs.html.erb